src/satosa/micro_services/account_linking.py

```diff
@@ -52,6 +53,10 @@ def _handle_al_response(self, context):
             satosa_logging(logger, logging.INFO, "issuer/id pair is linked in AL service",
                            context.state)
             internal_response.user_id = message
+            id_to_attr = self.config.get("id_to_attr", None)
```

Comment:
I'm only using `self.config = config` in order to get id_to_attr later on, so we even could do something like
```
self.id_to_attr = config.get("id_to_attr", None)
```